datalad/crawler/pipelines/fcptable.py

```diff
@@ -20,12 +19,19 @@
 
 
 def superdataset_pipeline(url=TOPURL):
+    """
+    Parameters
+    ----------
+    url: str
+       URL point to all datasets, hence the URL at the top
+    """
     annex = Annexificator()
     lgr.info("Creating a FCP collection pipeline")
     return [
         crawl_url(url),
         xpath_match('//*[@class="tableHdr"]/td/strong/text()', output='dataset'),
-        # TODO: replace spaces
+        # skipping Cleveland and NewYork due to URL redirects, Durham due to lack of dataset tarball
+        skip_if({'dataset': 'Cleveland CCF | Durham_Madden | NewYork_Test-Retest_Reliability'}, re=True),
```

Comment:
On Wed, 20 Jul 2016, Gergana Alteva wrote:
>    In datalad/crawler/pipelines/fcptable.py:
>  >      annex = Annexificator()
>  >      lgr.info("Creating a FCP collection pipeline")
>  >      return [
>  >          crawl_url(url),
>  >          xpath_match('//*[@class="tableHdr"]/td/strong/text()', output='dataset'),
>  > -        # TODO: replace spaces
>  > +        # skipping Cleveland and NewYork due to URL redirects, Durham due to lack of dataset tarball
>  > +        skip_if({'dataset': 'Cleveland CCF | Durham_Madden | NewYork_Test-Retest_Reliability'}, re=True),

>    delete spaces

and test (ok if just manually) if works as it should ;)